### PR TITLE
Add P&E Mailer tests for pe-mailer

### LINKED ISSUE #17: Add P&E Mailer Automation - Module
# 💡 Summary #

To further automate the delivery of P&E Reports to customers, a module/program will be implemented to retrieve pe_report output and deliver to the appropriate customer's email. Ideally this would be an in-line process that can be turned on/off will running the pe_reports module.

## Motivation and context ##

This would fully automate the P&E Reports delivery process. Secondarily allows for scalability and acquiring more customers at a faster pace. With proper test, it improves quality control.

## Implementation notes ##

- Proposed Architecture

```
├── src
│   ├── pe_reports
│   │   ├── __init__.py
│   │   ├── __main__.py
│   │   ├── _version.py
│   │   ├── data
│   │   ├── file_1.py
│   │   ├── file_2.py
│   │   └── report_generator.py
│   ├── pe_mailer
│   │   ├── __init__.py
│   │   ├── __main__.py
│   │   ├── _version.py
│   │   ├── data
│   │   ├── file_1.py
│   │   ├── file_2.py
│   │   └── mailer.py
└── tests
    ├── conftest.py
    └── test_pe_reports.py
    └── test_pe_mailer.py
├── setup.py
├── docker-compose/yml????
```
- Elegantly include secondary command line process for [mailer Usage](https://github.com/cisagov/cyhy-mailer).
- Update setup.py  to handle multiple `entry_points={'console_scripts':[ ]`
- Update dependancies for a python `import pe_mailer`

## Acceptance criteria ##

How do we know when this work is done?

- [x] TBD

### LINKED ISSUE #56: pe-mailer module "tests"
# 💡 Summary #

Add in "tests" for the pe-mailer module. These tests may include the following:
- test_message.py
- test_pandemessage.py
- test_reportmessage.py
- test_statsmessage.py

## Motivation and context ##
The tests are added for quality assurance and is standard practice for cisago/ repos.

## Implementation notes ##
- create appropriate scripts for pe-mailer; some derived from [cyhy-mailer test](https://github.com/cisagov/cyhy-mailer/tree/develop/tests)
- add to pe-reports/tests

## Acceptance criteria ##

How do we know when this work is done?

- [ ] Pass unittesting, pytest

### PR DESCRIPTION
**Pull request by schmelz21**
The P&E Reports module `pe-mailer` needs "tests" for better quality control.

## 🗣 Description ##

Where applicable, add pe-mailer test to`/tests`. Code referenced from the following standards : 
- [cisagov/skeleton-python-library/tests](https://github.com/cisagov/skeleton-python-library/tree/develop/tests)
- [cisagov/cyhy-mailer/tests](https://github.com/cisagov/cyhy-mailer/tree/develop/tests)

## 💭 Motivation and context ##

Meets expectations of the [cisagov code standards](https://github.com/cisagov/development-guide).

## 🧪 Testing ##

- All test PASS on run: `pytest tests/`

## ✅ Checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - _eschew scope creep!_
- [x] _All_ future TODOs are captured in issues, which are referenced
      in code comments.
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated
      to reflect the changes in this PR.
- [x] Tests have been added and/or modified to cover the changes in this PR.
- [x] All new and existing tests pass.

### CODE REVIEW

**Comment by schmelz21** [comment]
@cduhn17, @jsf9k, @dav3r 
I realized this should of been in draft mode while https://github.com/cisagov/pe-reports/pull/48 was pending approval. 
_I am sure its nice to finally see one of ours not so loaded with changes and limited in scope!_

When you have time, please review this PR - Thanks

**Review by jsf9k — APPROVED** [review]

File: src/pe_mailer/__init__.py [hunk 1]
```diff
@@ -1,3 +1,5 @@
 """The pe_mailer library."""
 
+from ._version import __version__  # noqa: F401
```
Comment (dav3r) [inline src/pe_mailer/__init__.py:6]: This should include our standard comment about why we have F401 here - see [here](https://github.com/cisagov/skeleton-python-library/blob/develop/src/example/__init__.py#L2-L6) for a comment you can steal.

File: tests/test_pe_mailer.py [hunk 2]
```diff
@@ -0,0 +1,101 @@
+"""File to be used to create test for pe-mailer module."""
```
Comment (dav3r) [inline tests/test_pe_mailer.py]: Prefer simpler wording:
```suggestion
"""Tests for the pe-mailer module."""
```

File: tests/test_pe_reports.py [hunk 3]
```diff
@@ -23,10 +23,11 @@
 PROJECT_VERSION = pe_reports.__version__
 
 
-# TODO: Replace current dummy test with useful tests - https://github.com/cisagov/pe-reports/issues/3#issue-909531010
+# TODO: Replace current dummy test with useful tests
+# Issue - https://github.com/cisagov/pe-reports/issues/3#issue-909531010
```
Comment (dav3r) [inline tests/test_pe_reports.py:27]: Can this TODO be removed now that there are better tests here?

File: tests/data/csv-sample.csv [hunk 4]
```diff
@@ -0,0 +1,2 @@
+0,1,2,A
+1,2,3,B
```
Comment (dav3r) [inline tests/data/csv-sample.csv]: Does this file serve any purpose?  I don't see it used in any of the tests.

**Review by dav3r — CHANGES_REQUESTED** [review]
Please take a look at my questions and suggestions.

File: tests/test_pe_reports.py (hunk 3 above)
Comment (schmelz21) [reply tests/test_pe_reports.py:27]: @dav3r, actually this one is in reference to just the` pe-report` module,  not the `pe-mailer` module. We do have intention to add in relevant tests once we get the `develop `branch updated with our new 1.0.0 report.

File: src/pe_mailer/__init__.py (hunk 1 above)
Comment (schmelz21) [reply src/pe_mailer/__init__.py:6]: Updated in d811d03

File: tests/data/csv-sample.csv (hunk 4 above)
Comment (schmelz21) [reply tests/data/csv-sample.csv]: @dav3r - nice catch.. this is irrelevant file for this pull-request.
File removed - dc78243

File: tests/test_pe_reports.py (hunk 3 above)
Comment (dav3r) [reply tests/test_pe_reports.py:27]: Got it- thanks for the clarification.

**Review by dav3r — APPROVED** [review]
This all LGTM now - thanks! 👍

**Pull request merged**